react/useGeolocationEnabled.tsx

```diff
@@ -0,0 +1,9 @@
+import { useQuery } from 'react-apollo'
+
+import QueryGeolocationInstalled from './QueryGeolocationInstalled'
+
+export default function useGeolocationEnabled() {
```

Comment:
Nit: how about using a simple export (without default)? My reasons are the same as the ones mentioned [here](https://basarat.gitbook.io/typescript/main-1/defaultisbad).

Reply:
we can't use simple exports from an IO app entrypoint

Reply:
just out of curiosity, this is a limitation with IO, right?

Reply:
yes